src/dojos/cd-dojo-details.vue

```diff
@@ -264,6 +285,12 @@
       padding: 45px 32px 32px 16px;
     }
 
+    &__section {
+      ~ .cd-dojo-details__section {
+        margin-top: 64px;
```

Comment:
Where is this 64px coming from ? Does it work in responsive ?

Reply:
It's from the desktop and mobile dojo details designs on Zeplin and is present on every section other than the top one.